docs/ocis/deployment/ocis_full.md

```diff
@@ -0,0 +1,353 @@
+  # S3 access key. Defaults to "ocis"
+  S3NG_ACCESS_KEY=
+  # S3 secret. Defaults to "ocis-secret-key"
+  S3NG_SECRET_KEY=
+  # S3 bucket. Defaults to "ocis"
+  S3NG_BUCKET=
+  # Add local minio S3 storage to the docker-compose file.
+  # This is needed for testing purposes.
+  # The leading colon is required to enable the service.
+  # S3NG_MINIO=:minio.yml
+  # Minio domain. Defaults to "minio.owncloud.test".
+  MINIO_DOMAIN=
+
+  # Define SMPT settings if you would like to send Infinite Scale email notifications.
+  # For more details see:
+  # https://doc.owncloud.com/ocis/latest/deployment/services/s-list/notifications.html
+  # NOTE: this doesn't work if you are using inbucket.
+  # SMTP host to connect to.
+  SMTP_HOST=
+  # Port of the SMTP host to connect to.
+  SMTP_PORT=
+  # An eMail address that is used for sending Infinite Scale notification eMails
+  # like "ocis notifications <noreply@yourdomain.com>".
+  SMTP_SENDER=
+  # Username for the SMTP host to connect to.
+  SMTP_USERNAME=
+  # Password for the SMTP host to connect to.
+  SMTP_PASSWORD=
+  # Authentication method for the SMTP communication.
+  SMTP_AUTHENTICATION=
+  # Allow insecure connections to the SMTP server. Defaults to false.
+  SMTP_INSECURE=
+
+  ## Default Enabled Services ##
+
+  ### Apache Tika Content Analysis Toolkit ###
+  # Tika (search) is enabled by default, comment if not required.
+  # The leading colon is required to enable the service.
+  TIKA=:tika.yml
+  # Set the desired docker image tag or digest.
+  # Defaults to "latest"
+  TIKA_IMAGE=
+
+  ### Collabora Settings ###
+  # Collabora web office is default enabled, comment if not required.
+  # The leading colon is required to enable the service.
+  COLLABORA=:collabora.yml
+  # Domain of Collabora, where you can find the frontend.
+  # Defaults to "collabora.owncloud.test"
+  COLLABORA_DOMAIN=
+  # Domain of the wopiserver which handles OnlyOffice.
+  #Defaults to "wopiserver.owncloud.test"
+  WOPISERVER_DOMAIN=
+  # Admin user for Collabora.
+  # Defaults to "admin".
+  # Collabora Admin Panel URL:
+  # https://{COLLABORA_DOMAIN}/browser/dist/admin/admin.html
+  COLLABORA_ADMIN_USER=
+  # Admin password for Collabora.
+  # Defaults to "admin".
+  COLLABORA_ADMIN_PASSWORD=
+  ...
+  ```
+  #### Reverse Proxy and SSL
+
+  {{< hint type=important >}}
+  **Domains and SSL**\
+  Though it may sound strange, most of the setups are failing due to a misconfiguration regarding domains and SSL. Please make sure that you have set up the domains correctly and that they are pointing to your server. Also, make sure that you have set up the email address for the LetsEncrypt certificates in `TRAEFIK_ACME_MAIL=`.
+  {{< /hint >}}
+
+  You are installing oCIS on a server and Traefik will obtain valid certificates for you so please remove `INSECURE=true` or set it to `false`.
+
+  Traefik will issue certificates with LetsEncrypt and therefore you must set an email address in `TRAEFIK_ACME_MAIL=`.
+
```

Comment:
can u mention that the consumer has to accept all the following certs manually.

```
- keycloak.owncloud.test
- ocis.owncloud.test
- traefik.owncloud.test
- collabora.owncloud.test
- onlyoffice.owncloud.test
- wopiserver.owncloud.test
- mail.owncloud.test
- companion.owncloud.test
- minio.owncloud.test
```

Reply:
It is in line 243.